### PR TITLE
MODKBEKBJ-696 Load Holdings: NullPointerException while reading started date

### PR DESCRIPTION
**Pull request by Shans-Kaluhin**
### Learning
https://issues.folio.org/browse/MODKBEKBJ-696

### CODE REVIEW

File: src/main/java/org/folio/service/holdings/HoldingsServiceImpl.java [hunk 1]
```diff
@@ -328,7 +328,11 @@ private boolean canChangeStatus(HoldingsLoadingStatus status) {
   }
 
   private boolean isHangedLoading(HoldingsLoadingStatus status) {
-    OffsetDateTime startDate = getZonedDateTime(status.getData().getAttributes().getStarted());
+    String started = status.getData().getAttributes().getStarted();
```
Comment (sherzodn) [inline src/main/java/org/folio/service/holdings/HoldingsServiceImpl.java:331]:
```suggestion
    var started = status.getData().getAttributes().getStarted();
```

File: src/main/java/org/folio/service/holdings/HoldingsServiceImpl.java [hunk 2]
```diff
@@ -328,7 +328,11 @@ private boolean canChangeStatus(HoldingsLoadingStatus status) {
...
+    if (StringUtils.isEmpty(started)) {
+      return true;
+    }
+    OffsetDateTime startDate = getZonedDateTime(started);
```
Comment (sherzodn) [inline src/main/java/org/folio/service/holdings/HoldingsServiceImpl.java:335]:
```suggestion
    var startDate = getZonedDateTime(started);
```

File: src/test/java/org/folio/util/HoldingsStatusUtil.java [hunk 3]
```diff
@@ -36,7 +36,6 @@ public static HoldingsLoadingStatus saveStatus(String credentialsId, HoldingsLoa
                                                  OffsetDateTime startedTime, Vertx vertx) {
     CompletableFuture<HoldingsLoadingStatus> future = new CompletableFuture<>();
     String query = DbUtil.prepareQuery(insertLoadingStatus(), holdingsStatusTestTable(), createPlaceholders(4));
-    status.getData().getAttributes().setStarted(startedTime.format(DateTimeFormatter.ISO_OFFSET_DATE_TIME));
```
Comment (sherzodn) [inline src/test/java/org/folio/util/HoldingsStatusUtil.java:39]: Remove unused import statement from 21st line, causing pipeline to fail

Comment (Shans-Kaluhin) [reply src/test/java/org/folio/util/HoldingsStatusUtil.java:39]: Thanks! ✔️

**Review by psmagin — APPROVED** [review]

**Review by viacheslavkol — APPROVED** [review]

**Pull request merged**